### PR TITLE
OC-258_Migrate_microgrids_functionality_to_new_message_processing_solution

### PR DESCRIPTION
**Pull request by riannefijn**

### CODE REVIEW

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 1]
```diff
@@ -85,57 +90,31 @@ jms.default.use.exponential.back.off=true
...
-# --- CONCURRENT CONSUMERS ---
-jms.microgrids.requests.concurrent.consumers=2
-jms.microgrids.requests.max.concurrent.consumers=10
-
 # --- REDELIVERY POLICY ---
-jms.microgrids.requests.maximum.redeliveries=3
-# Set initial redelivery delay in ms (60000 = 1 min)
-jms.microgrids.requests.initial.redelivery.delay=60000
-# Set redelivery delay in ms (60000 = 1 min)
 jms.microgrids.requests.redelivery.delay=60000
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Replace by default properties to reduce number of properties?

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 2]
```diff
@@ -85,57 +90,31 @@ jms.default.use.exponential.back.off=true
...
-# Set maximum redelivery delay in ms (300000 ms = 5 min)
 jms.microgrids.requests.maximum.redelivery.delay=300000
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Replace by default properties to reduce number of properties?

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 3]
```diff
@@ -85,57 +90,31 @@ jms.default.use.exponential.back.off=true
...
-jms.microgrids.requests.back.off.multiplier=2
-jms.microgrids.requests.use.exponential.back.off=true
 
+# --- CONCURRENT CONSUMERS ---
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Replace by default properties to reduce number of properties?

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 4]
```diff
@@ -85,57 +90,31 @@ jms.default.use.exponential.back.off=true
...
-# Set explicitQosEnabled to true to enable the use of deliveryMode, priority, and timeToLive
-jms.microgrids.responses.explicit.qos.enabled=true
-# Set delivery persistent
-jms.microgrids.responses.delivery.persistent=true
-# Set Time to live in ms (1800000 = 30 minutes)
-jms.microgrids.responses.time.to.live=3600000
-# Set receive timeout
 jms.microgrids.responses.receive.timeout=100
 
 # ---REDELIVERY POLICY ---
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Replace by default properties to reduce number of properties?

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 5]
```diff
@@ -85,57 +90,31 @@ jms.default.use.exponential.back.off=true
...
-# Set initial redelivery delay in ms (60000 = 1 min)
 jms.microgrids.responses.initial.redelivery.delay=60000
-# Set redelivery delay in ms (60000 = 1 min)
 jms.microgrids.responses.redelivery.delay=60000
-# Set maximum redelivery delay in ms (300000 = 5 min)
 jms.microgrids.responses.maximum.redelivery.delay=300000
-jms.microgrids.responses.back.off.multiplier=2
-jms.microgrids.responses.use.exponential.back.off=true
+
+# --- CONCURRENT CONSUMERS ---
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Replace by default properties to reduce number of properties?

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 6]
```diff
@@ -145,22 +124,10 @@ jms.microgrids.responses.use.exponential.back.off=true
...
-# Set explicitQosEnabled to true to enable the use of deliveryMode, priority, and timeToLive
-jms.microgrids.logging.explicit.qos.enabled=true
-# Set delivery persistent
-jms.microgrids.logging.delivery.persistent=true
-# Set Time to live in ms (180000 = 3 minutes)
-jms.microgrids.logging.time.to.live=3600000
-# Set receive timeout
 jms.microgrids.logging.receive.timeout=10
 
 # --- REDELIVERY POLICY ---
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Replace by default properties to reduce number of properties?

File: osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties [hunk 7]
```diff
@@ -85,57 +90,31 @@ jms.default.use.exponential.back.off=true
...
-jms.microgrids.requests.time.to.live=3600000
-
-# --- CONCURRENT CONSUMERS ---
-jms.microgrids.requests.concurrent.consumers=2
-jms.microgrids.requests.max.concurrent.consumers=10
-
 # --- REDELIVERY POLICY ---
-jms.microgrids.requests.maximum.redeliveries=3
-# Set initial redelivery delay in ms (60000 = 1 min)
-jms.microgrids.requests.initial.redelivery.delay=60000
```
Comment (smvdheijden) [inline osgp-adapter-ws-microgrids/src/main/resources/osgp-adapter-ws-microgrids.properties]: Keep this

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Platform?pullRequest=558) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: riannefijn<br/>:x: web-flow

**Pull request merged**